### ISSUE TITLE
Jig init not working

### ISSUE THREAD

**Issue by zmarouf** [robmadole/jig#11]
`$ jig init .`
                                                                                                                                                                          
usage: jig [-h] COMMAND

optional arguments:
  -h, --help  show this help message and exit

jig commands:
  config      Manage settings for installed Jig plugins
  install     Install a list of Jig plugins from a file
  plugin      Manage this repository's Jig plugins
  report      Run plugins on a revision range
  runnow      Run plugins on staged changes and show the results
  version     Show Jig's version number

See `jig COMMAND --help` for more information
___

Expected:
Git repository has been initialized for use with Jig.

You should tell Git to ignore the new .jig directory. Run this:

    $ echo ".jig" >> .gitignore

Next install some plugins. Jig has a common set you may like:

    $ curl -L https://raw.github.com/robmadole/jig-plugins/lists/common.txt > .jigplugins.txt
    $ jig install .jigplugins.txt
___
Pointers?

**Comment by robmadole** [comment]
Interesting. It looks like the `init` subcommand is not available. Can you tell me some information about your system? Python and Git version? A quick `python --version` and `git --version` should tell ya.

**Comment by zmarouf** [comment]
Sorry for the late reply I was swamped with work.
I'm running OSX Yosemite with the following versions of the 2 executables. 

`$ python --version`                                                                                                                                                                         
Python 2.7.6

`$ git --version`                                                                                                                                                                             
git version 2.2.2
---
I asked a colleague of mine to check earlier and he had the same issue on his Ubuntu 14.04. 
Hope this helps a little :)

**Comment by robmadole** [comment]
Thanks for the details. I'll check this out on Ubuntu 14.04 and figure out what's going on. Sorry for the hassle!

**Comment by tomleo** [comment]
Having the same problem:

```
~/$ jig init .
usage: jig [-h] COMMAND

optional arguments:
  -h, --help  show this help message and exit

jig commands:
  report      Run plugins on a revision range
  runnow      Run plugins on staged changes and show the results
  plugin      Manage this repository's Jig plugins
  config      Manage settings for installed Jig plugins
  version     Show Jig's version number
  install     Install a list of Jig plugins from a file

See `jig COMMAND --help` for more information
```

System Info:

```
pip freeze | grep -i jig
jig==0.1.10

python --version
Python 2.7.8

git --version
git version 2.1.0

cat /etc/lsb-release 
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=14.10
DISTRIB_CODENAME=utopic
DISTRIB_DESCRIPTION="Ubuntu 14.10"
```

**Comment by robmadole** [comment]
thanks @tomleo. I have some time to look at this tomorrow. I'll see if I can get it fixed.

**Comment by robmadole** [comment]
@tomleo @zmarouf I've just released 0.1.11 which fixes this issue. I tested this on Ubuntu 14. Give it a shot and if you still have problems let me know and I'll re-open this issue. Thank you both for letting me know!
